cmd/loop/loopout.go

```diff
@@ -113,6 +120,7 @@ func loopOut(ctx *cli.Context) error {
 		MaxSwapRoutingFee:   int64(*limits.maxSwapRoutingFee),
 		LoopOutChannel:      unchargeChannel,
 		SweepConfTarget:     sweepConfTarget,
+		MaxSwapWaitTimeSec:  ctx.Uint64("max_swap_wait_time"),
```

Comment:
This is where we'd convert the relative deadline from the cmd line to an absolute time.